abi-gen/templates/TypeScript/contract.handlebars

```diff
@@ -381,9 +385,9 @@ export class {{contractName}}Contract extends BaseContract {
         txDefaults?: Partial<TxData>,
         logDecodeDependencies?: { [contractName: string]: ContractAbi },
         deployedBytecode: string | undefined = {{contractName}}Contract.deployedBytecode,
-        encodingRules?: EncodingRules,
+        encodingOpts?: EncodingRules | Partial<EncoderOverrides>,
```

Comment:
There's exactly one place in the world where we would ever pass in this final constructor arg, right? Why not just always have it be `EncoderOverrides`?

Reply:
Agreed, left it in for a/b tests. GET BE GONE